plugins/modules/redfish_command.py

```diff
@@ -239,8 +239,19 @@
     type: bool
     default: false
     version_added: 3.7.0
+  bios_attributes:
+    required: false
+    description:
+      - BIOS attributes that needs to be verified in the given server
+    type: dict
+    version_added: 6.1.0
 
-author: "Jose Delarosa (@jose-delarosa)"
+author:
+  - "Jose Delarosa (@jose-delarosa)"
+  - "Gayathiri Devi Ramasamy (@Gayathirideviramasamy)"
+  - "T S Kushal (@TSKushal)"
+  - "Varni H P (@varini-hp)"
+  - "Prativa Nayak (@prativa-n)"
```

Comment:
Same question here as in #5899.

Reply:
Resolving this conversation, as we discussed and closed this in PR [5899](https://github.com/ansible-collections/community.general/pull/5899). Thanks!